src/components/Text.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, PropsWithChildren, ReactNode } from 'react';
+import { Typography, TypographyProps } from '@mui/material';
+
+interface CustomTypographyProps extends TypographyProps {
+  children: PropsWithChildren<ReactNode>;
+  fontSize?: string | number;
+}
+
+const Text: FC<CustomTypographyProps> = ({
```

Comment:
no FC

Reply:
not a blocker though. I can figure out the correct typing without using FC. We shouldn't be using FC